### DISCUSSION TITLE
Wio-E5 mini | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#79] [Q&A]
# Wio-E5 mini | Seeed Studio Wiki

-->

https://wiki.seeedstudio.com/LoRa_E5_mini/

**Comment by srFelipes** [comment]
Hi! I wanted to ask about the pinout differences between the GPIO of NUCLEO-WL55JC and the WIO-E5 particularly about the sck pin that connect to pb13 and the boot button, does this mean that the boot button cannot be used if the SPI port is used?.
Thanks in advance

**Comment by RadioSteve656** [comment]
Hi, When I download the Github repo the .project files does not show up. Has anyone else had this problem. How do you solve it?

**Reply by RadioSteve656** [reply]
I have also tried downloading them individually and adding them to the folder, but that too does not work. CubeIDE still says that there are missing files and information.

**Reply by tonyleo723** [reply]
The **.project** files are listed in the example directory. For example in the **/Projects/Applications/LoRaWAN/LoRaWAN_End_Node/STM32CubeIDE/** you will find the exact **.cproject** files for this example.

Or maybe your operating system recognises these files as hidden? (e.g. on Linux)

**Reply by RadioSteve656** [reply]
Yep, the problem was my linux machine. All working now! Thanks!

**Comment by dyane6** [comment]
Hello seeed studio team,

Please could you upgrade your software with the last version of STM32CUBE MX and STM32CUBE IDE?
Best regards.
Olivier

**Comment by dyane6** [comment]
Hello seeed studio team,

Please could you upgrade your softwares with the last version of STM32CUBE MX and STM32CUBE IDE?
Because the files .ioc of your examples are not compliant anymore with the new ST Software Development Tools.
Best regards.
Olivier

**Reply by Jessie219-web** [reply]
there is currently no update plan, but we will pass your feedback to the team

**Comment by dyane6** [comment]
Updates of examples would be appreciated. Thanks.

**Comment by pkuppusamy77** [comment]
What is the physical dimensions of E5-Mini-Dev Board, there is no datasheet for it..

**Reply by DavidDu-Seeed** [reply]
The development board measures 50 × 23 mm, excluding the SMA antenna connector. You can also visit the link below to view more product details on our product page: https://www.seeedstudio.com/LoRa-E5-mini-STM32WLE5JC-p-4869.html
